content/versions.json

```diff
@@ -34,7 +34,8 @@
     "1.4.1",
     "1.4.2",
     "1.5.0",
-    "1.5.1"
+    "1.5.1",
+    "1.5.2"
```

Comment:
This change is a Minor release as we've added new functionality so will be 1.6.0 rather than 1.5.2

As a general note, we use semantic versioning with Hexagon and follow the below pattern:
- *Major*: Any breaking api changes (e.g. removal of methods or modules)
- *Minor*: Non-breaking api changes (eg new features or deprecations)
- *Patch*: Bugfixes